src/buffer/h264.rs

```diff
@@ -194,9 +194,13 @@ impl PictureParameterBufferH264 {
 }
 
 /// Wrapper over the `VASliceParameterBufferH264` FFI type.
-pub struct SliceParameterBufferH264(Box<bindings::VASliceParameterBufferH264>);
+pub struct SliceParameterBufferH264(Vec<bindings::VASliceParameterBufferH264>);
```

Comment:
Btw, why is a vector needed at all here? We only ever add or access a single element, and even if `add_slice_parameter` is called several times, `Buffer::new` will only return the size of a single `VASliceParameterBufferH264` as its total size. Or maybe this is a bug?

Reply:
Ah, looking at https://github.com/chromeos/cros-codecs/pull/66/commits/5851cfca68d986a5f408e410a4fff5a0b2e59e57 it is now obvious why this is needed - never mind!